test/metabase/driver/common/parameters/operators_test.clj

```diff
@@ -43,7 +43,34 @@
                                             26
                                             {:source-field 5}]]
                                   :value  ["foo"]})
-           [:starts-with [:field 26 {:source-field 5}] "foo"]))))
+           [:starts-with [:field 26 {:source-field 5}] "foo"]))
+    (testing "with options"
+      (is (= (params.ops/to-clause {:type   :string/starts-with
+                                    :target [:dimension
+                                             [:field
+                                              26
+                                              {:source-field 5}]]
+                                    :value  ["foo"]
+                                    :options {:case-insensitive false}})
+             [:starts-with [:field 26 {:source-field 5}] "foo" {:case-insensitive false}])))
+    (testing "with multiple arguments"
```

Comment:
Can you please make these separate tests? It's super hard to debug test failures if we have a bunch of mega tests that each have like 100 assertions. You can't rerun a single logical test case independently of all the other ones if they're all lumped into a giant test

I just went in and split that giant `to-clause-test` up into 6 different tests a few months ago, please don't make me have to go in there and do it again